### DISCUSSION TITLE
fast server not working

### DISCUSSION THREAD

**Discussion by moreshubham007** [postalserver/postal#2059] [Help with using Postal]
I am trying to enable fast server 
this is the configuration i used  
fast_server:
  enabled: true
  bind_address: 172.20.20.13
  port: 4000
 on postal ui it shows like its enable but when i click on tracking links it fails to open 
i added 172.20.20.13:4000 to nginx proxy manager its aslo fails to configure 

please help with how to configure fast server

**Comment by willpower232** [comment]
fast server is no longer enabled via the config file. It no longer has a separate port as it no longer manages its own certificates.

Please have another look at the documentation https://docs.postalserver.io/features/click-and-open-tracking and particularly

> You'll need to add an appropriate virtual host on your web server that proxies traffic from that domain to the Postal web server. The web server must add the `X-Postal-Track-Host: 1` header so the Postal web server knows to treat requests as tracking requests.

**Reply by moreshubham007** [reply]
thank you for your response 
i have added header on nginx proxy manager but its not working 

![image](https://user-images.githubusercontent.com/48375001/177990063-7c387661-c8c7-4db4-b6bd-4324cc4c2cfe.png)
this is the tracking link 

http://emall.test.teamlogicwoodbridge.com/inybjx/yBgZM48O
![image](https://user-images.githubusercontent.com/48375001/177990928-4b3a4ac1-9b5b-4125-93b2-29310b1dd2fa.png)

i added CNAME record 
for emall.test.teamlogicwoodbridge.com for below 
 track_domain: mail.teamlogicwoodbridge.comtracing link

**Reply by willpower232** [reply]
have a look at #1693 where they apparently figured out how to use nginx proxy manager
